GateAdventures/Engine/Scenes/Scene.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using GateAdventures.Engine.Scenes;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace GateAdventures;
+
+/// <summary>
+/// A base for most scene implementations.
+/// </summary>
+public abstract class Scene : IScene
+{
+	protected GraphicsDeviceManager GraphicsDeviceManager { get; private set; }
```

Comment:
Especially in downstream classes, i.e. HelloScene.cs, etc, it might be more clear that it's an instance variable with camel case, rather than think it's something like an imported class

Reply:
I rescind this

Reply:
Camel case is reserved for method body variables or params based on C# guidelines. I added underscores to these instead